lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "lerna": "2.5.1",
+  "version": "6.0.0",
```

Comment:
could use independent versioning, but not biggie

Reply:
Was intentional to not use independent.  😄 

If you have some good reading on why independent is preferable, I'm happy to read and reevaluate on the next release.

Reply:
Short of it is that even though you make a breaking change in the react config, there's no need to make a breaking change in all of the other packages

Reply:
Understood.  That was a consideration when choosing but my preference is fixed versions.  Inspecting a package.json it's easy to tell when you have compatible versions.  This isn't an issue if you're on the latest versions, but if you're a few behind you have more effort to look up which package goes with which.